src/reconcile/rec_write.c

```diff
@@ -3172,15 +3128,112 @@ __rec_split_raw(WT_SESSION_IMPL *session, WT_RECONCILE *r, size_t next_len)
 }
 
 /*
+ * __rec_split_finish_process_prev --
+ * 	If the two split chunks together fit in a single page, merge them into
+ * 	one. If they do not fit in a single page but the last is smaller than
+ * 	the minimum desired, move some data from the penultimate chunk to the
+ * 	last chunk and write out the previous/penultimate. Finally, update the
+ * 	pointer to the current image buffer.
```

Comment:
After this function exits, we will only have one buffer, the current image pointer. At the end of the function:
r->cur_img_ptr = r->prev_img_ptr;

I will update comment to address that expectation is to write out of current image pointer after this function exits